spec/namespaces/cat.yaml

```diff
@@ -842,11 +843,46 @@ components:
     cat.help@200:
       description: ''
       content:
-        application/json:
+        text/plain:
```

Comment:
Created an issue for this: https://github.com/opensearch-project/opensearch-api-specification/issues/293

Reply:
Thanks! Tests caught the bug. Which is good news.